bfasst/yaml_parser.py

```diff
@@ -63,7 +60,7 @@ def __collect_design_paths(self):
                         continue
 
                     if (design_child / "design.yaml").is_file():
-                        design_name = design_child.split("/")[-2:]
+                        design_name = "/".join(str(design_child).split("/")[-2:])
```

Comment:
This seems to be removing the first to items of the path?  What's the reason for this?  

It seems like you might want to use `relpath`, relative to something?

Or if you really want to always remove the first to elements, add a comment explaining why, and there is a better approach here: https://stackoverflow.com/a/26724369/609215

Reply:
Basically, this is because all the ninja stuff right now expects to take the design in as a subdirectory of `designs/`. This reverts the change Ramez made a few weeks ago and is undesirable, but it is in line with the current structure of our CI yaml experiments. Reilly and I have already talked about this, and we want to make the parser require the full relative path to each design we want to run. This is planned for a future pr. I was trying to get to it this week, but have been really bogged down by rapidwright stuff.

Reply:
The CI checks right now specify something like:
```
designs:
    - byu/alu
```
And we want to make it require
```
designs:
    - designs/byu/alu
```
Or some other complete path relative to where the script is invoked. This will allow designs to reside outside the bfasst repo. The bfasster script allows multiple designs to be passed by yaml or a single design to be passed by cmd line argument, so the format for the designs in both cases needs to match. I initially made the format match what was already in the CI yaml files.

Reply:
I just made the requested changes, in the interim.